package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@epages/react-components",
   "description": "A zoo of useful React components.",
-  "version": "0.10.3",
+  "version": "0.10.4",
```

This is a breaking change, please increase the minor version
```suggestion
  "version": "0.11.0",
```